Relax the atol/rtol of layernorm math kernel test.

ailzhang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#49507 Relax the atol/rtol of layernorm math kernel test.**

Differential Revision: [D25598424](https://our.internmc.facebook.com/intern/diff/D25598424)

Commit:
Relax the atol/rtol of layernorm math kernel test.

[ghstack-poisoned]

mruberry reviewed (approved):
Cool! It's too bad we don't have a Python exposure for this or the nicer error messages that our Python tests print. That would help us understand the failure.

ailzhang commented:
@mruberry Good point! Yea let me add some messages before we land this...

Commit:
Update on "Relax the atol/rtol of layernorm math kernel test."


Differential Revision: [D25598424](https://our.internmc.facebook.com/intern/diff/D25598424)

[ghstack-poisoned]

mruberry commented on aten/src/ATen/test/math_kernel_test.cpp:
```diff
@@ -4,9 +4,22 @@
+  bool equal = t1.allclose(t2, rtol, atol);
+  if (!equal) {
+    std::cerr << "Difference in tensor value: \nFirst tensor:\n"
+        << t1 << "\nSecond tensor:\n" << t2 << std::endl;
```
This is cool, although it may be hard to spot where the tensors are different. Still seems like an improvement, especially when t1 and t2 are small.

ailzhang replied:
Yea I think having max difference and their location is useful in local debugging, but it's generally more useful to have the full tensor available when shown on CI(especially when it comes to setting atol/rtol). Let's try this first, if it needs more detailed inforation later I can add them too :D
